app-collab-note/src/App.css

```diff
@@ -5,7 +5,7 @@
 }
 
 .App {
-	font-family: 'Open Sans', sans-serif;
+	  font-family: 'Open Sans', sans-serif;
```

Comment:
is this change made by the linter? looks strange to me

Reply:
I might have formatted the document in vscode.😅

Reply:
looks like it was just one line, not the whole doc

Reply:
Sorry, I didn't intend for it to be in the commit. Changed it now.